CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
-# 4.9.1 - 2021/02/15
+# 4.10.0 - 2021/02/16
```

Comment:
This looks like we're losing the 4.9.1 changelog

Reply:
Well spotted, seems I hashed the rebase.